mobile/src/main/java/org/hawkular/client/android/service/AlertService.java

```diff
@@ -34,8 +34,8 @@
     Call<List<Alert>> get(@QueryMap Map<String, String> parameters);
 
     @FormUrlEncoded
-    @POST("/hawkular/alerts/ack/{alertId}")
```

Comment:
This one is correct api for call for ack alert as below one do not provide {alertId}

major difference between aerogear pipe and retrofit in case of POST and PUT is, in case of pipe we need not to put the id of item we are sending on POST as it was resolved from recordid of item we are sending. But in case of retrofit we need to configure that by ourself